src/uu/head/src/head.rs

```diff
@@ -238,11 +238,18 @@ impl HeadOptions {
             Some(v) => v.cloned().collect(),
             None => vec!["-".to_owned()],
         };
-        //println!("{:#?}", options);
+
         Ok(options)
     }
 }
 
+fn map_stdout_error(err: io::Error) -> io::Error {
```

Comment:
The function name is a bit misleading as the function "only" creates a new error and doesn't do any mapping.

Reply:
Also, maybe we could `#[inline]` it ?